### DISCUSSION TITLE
Log in to Trakt failed

### DISCUSSION THREAD

**Discussion by shotfirer** [Taxel/PlexTraktSync#1292] [Support] [answered]
I'm setting up the app for the first time, and can't authenticate with Trakt. I added it to Trakt OAuth apps and got the client ID and secret (I removed them from the log).
```
Please enter your client id: 7305...                            
Please enter your client secret: 2bcd...
Attempting to authenticate with Trakt
Log in to Trakt failed: Expecting value: line 1 column 1 (char 0), Try again.
Please enter your client id: 
```

**Comment by scoiattolo49** [comment, marked as answer]
Trakt is down since a few days ago: they had a massive database failure. They are in the process of recovering but some data may be permanently lost (namely last month).

Trakt api will be hopefully restored next week, until then Plextraktsync is not going to work.

**Reply by ChristoffBo** [reply]
same issue

**Comment by scoiattolo49** [comment]
Trakt api is now back online, so Plextraktsync is working again as intended.

I didn’t lose any data and the sync updated Trakt for the last 10 days when Trackt was off
